shell.nix

```diff
@@ -35,8 +45,46 @@ in pkgs.mkShell {
   LANGUAGE = "en_US.UTF-8";
 
   QTDIR = qtCustom;
+  # TODO: still needed?
+  # https://github.com/NixOS/nixpkgs/pull/109649
+  QT_INSTALL_PLUGINS = "${qtCustom}/${pkgs.qt515.qtbase.qtPluginPrefix}";
 
   shellHook = ''
-    export MAKEFLAGS="-j$NIX_BUILD_CORES"
```

Comment:
@cammellos, are you ok if I remove this line?
`shell.nix` will be used in Jenkins too, which will need another jobs count.
(currently 12 cores / 3 executors = 4 jobs)

We could have some logic for jobs cound in the future.
You will be still able to set `MAKEFLAGS` in you env before running `nix-shell`

Reply:
UPD: can keep it, no prob for CI

Reply:
Also: `EXECUTOR_NUMBER`, https://ci.status.im/env-vars.html/

Reply:
Using `NIX_BUILD_CORES` should work well.